src/components/DopplerPlus/UpgradeSuggestionForm.test.js

```diff
@@ -45,165 +45,160 @@ describe('UpgradeSuggestionForm component', () => {
+    const inputLastname = screen.getByRole('textbox', { name: 'signup.label_lastname' });
+    fireEvent.change(inputLastname, { target: { value: '' } });
 
-      const inputPhone = container.querySelector('input#phone');
-      fireEvent.change(inputPhone, { target: { value: '' } });
-    });
+    const inputPhone = screen.getByRole('textbox', { name: 'signup.label_phone' });
+    fireEvent.change(inputPhone, { target: { value: '' } });
 
+    // Act
     await act(async () => {
-      const submitButton = container.querySelector('button[type="submit"]');
+      const submitButton = await screen.findByRole('button', { name: /submit/i });
       fireEvent.submit(submitButton);
     });
 
     // Assert
-    expect(getAllByText('validation_messages.error_required_field').length).toBe(4);
+    expect(screen.getAllByText('validation_messages.error_required_field').length).toBe(4);
   });
 
   it('should show error message if email field is empty', async () => {
     // Arrange
-    const { container, getByText } = render(<UpgradeSuggestionFormElement />);
+    render(<UpgradeSuggestionFormElement />);
 
-    // Act
-    act(() => {
-      const inputEmail = container.querySelector('input#email');
-      fireEvent.change(inputEmail, { target: { value: '' } });
+    const inputEmail = screen.getByRole('textbox', { name: 'signup.label_email' });
+    fireEvent.change(inputEmail, { target: { value: '' } });
 
-      const inputName = container.querySelector('input#name');
-      fireEvent.change(inputName, { target: { value: 'Juan' } });
+    const inputName = screen.getByRole('textbox', { name: 'signup.label_firstname' });
+    fireEvent.change(inputName, { target: { value: 'Juan' } });
 
-      const inputLastname = container.querySelector('input#lastname');
-      fireEvent.change(inputLastname, { target: { value: 'Perez' } });
+    const inputLastname = screen.getByRole('textbox', { name: 'signup.label_lastname' });
+    fireEvent.change(inputLastname, { target: { value: 'Perez' } });
 
-      const inputPhone = container.querySelector('input#phone');
-      fireEvent.change(inputPhone, { target: { value: '+54 223 655-8877' } });
-    });
+    const inputPhone = screen.getByRole('textbox', { name: 'signup.label_phone' });
+    fireEvent.change(inputPhone, { target: { value: '+54 223 655-8877' } });
 
-    act(() => {
-      const submitButton = container.querySelector('button[type="submit"]');
+    // Act
+    await act(async () => {
+      const submitButton = await screen.findByRole('button', { name: /submit/i });
       fireEvent.submit(submitButton);
     });
 
     // Assert
     await waitFor(() => {
-      return expect(getByText('validation_messages.error_required_field')).toBeInTheDocument();
+      return expect(
+        screen.getByText('validation_messages.error_required_field'),
+      ).toBeInTheDocument();
     });
   });
 
   it('should show error message if name field is empty', async () => {
```

Comment:
This test shouldn't be here anymore right? It is replaced by `'should show error message if email field is empty'`...

Reply:
But this changes could be done in another PR so this is not an infinite PR.